src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,137 @@
+package com.epam.tc.hw2.ex1;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class Exercise1 {
+
+    private WebDriver webDriver;
+    private WebElement webElement;
+    private final String address = "https://jdi-testing.github.io/jdi-light/index.html ";
+
+    private final String login = "Roman";
+    private final String password = "Jdi1234";
+    private final String username = "ROMAN IOVLEV";
```

Comment:
Let's create BaseTest class where you can move this constants, driver initialization and beforeClass, afterClass methods

Reply:
BaseTest.java created